daemon/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pstas/libindy:v1.6.7
+FROM pstas/libindy:v1.6.6
```

Comment:
@norus I didn't have to do this, maybe this is remnant code? I would expect the actual build of your libindy on this line: INDY_VERSION="v1.6.7": https://github.com/Patrik-Stas/indyscan/blob/2b074295469b423e9f64c62737e7615508119448/libindy-ubuntu/build.sh#L3 to need to be changed as well

Reply:
There was no 1.6.7 tag, only 1.6.6: https://hub.docker.com/r/pstas/libindy/tags

Reply:
Ahh my apologies, you're right. 
I had just built the docker image inside of `/libindy-ubuntu` locally and tagged it as `pstas/lbindy:v1.6.7.  Good catch!  Perhaps build-local should also not rely on the DockerHub if it's entirely local.

Reply:
Yes, I thought about that too, hence added "quick" in my description ;)

I suppose libindy could be built locally as well.

Reply:
Thanks for heads up guys. I've pushed the v1.6.7 to dockerhub https://cloud.docker.com/u/pstas/repository/docker/pstas/libindy 
So building the containers locally should work now. 

Ideally it should be possible to easily build all containers without any dependency on dockerhub. Though I suppose I'll keep pushing them there, because build of the libindy container can take quite some time.